src/components/App.js

```diff
@@ -8,21 +8,14 @@ import { auth } from 'util/firebase';
 import toJS from 'util/to-js';
 import { CURRENT_TERM } from 'util/constants';
 import PropTypes from 'prop-types';
-import { withStyles } from '@material-ui/styles';
 import { withRouter, Route } from 'react-router-dom';
 import TopBar from './TopBar';
 import NavDrawer from './NavDrawer';
-import HomePage from './Pages/HomePage';
-import AboutPage from './Pages/AboutPage';
-import FAQPage from './Pages/FAQPage';
-import BugReportPage from './Pages/BugReportPage';
-import ContactPage from './Pages/ContactPage';
-
-export const styles = {
```

Comment:
When the actual page contents of App were moved to `HomePage` (which is now `AppBody`), the styles were not moved with it. This fixes that.